docs/tutorials/HD143006_Part_2.py

```diff
@@ -0,0 +1,481 @@
+
+# + cell_id="00001-be94721b-eee2-4e2e-96e2-dc65b9fd4f5b" deepnote_cell_type="code" deepnote_to_be_reexecuted=false execution_millis=663 execution_start=1623447169390 source_hash="4f0f20f8" tags=[]
+from astropy.io import fits
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy.utils.data import download_file
+from torch.utils import tensorboard
+
+# downloading fits file
+fname_F = download_file(
+    "https://almascience.nrao.edu/almadata/lp/DSHARP/images/HD143006_continuum.fits",
+    cache=True,
+    pkgname="mpol",
+)
+# downloading extracted visibilities file
+fname_EV = download_file(
+    "https://zenodo.org/record/4904794/files/HD143006_continuum.npz",
+    cache=True,
+    pkgname="mpol",
+)
+# load extracted visibilities from npz file
+dnpz = np.load(fname_EV)
+uu = dnpz["uu"]
+vv = dnpz["vv"]
+weight = dnpz["weight"]
+data = dnpz["data"]
+# opening the fits file
+dfits = fits.open(fname_F)
+cdelt_scaling = dfits[0].header["CDELT1"] * 3600  # scaling [arcsec]
+cell_size = abs(cdelt_scaling)  # [arcsec]
+# close fits file
+dfits.close()
+
+# + cell_id="00008-d56e3afe-45cf-4fe5-a4e7-1803f28deec4" deepnote_cell_type="code" deepnote_to_be_reexecuted=false execution_millis=27 execution_start=1623441758279 source_hash="b76fed2d"
+from mpol import gridding, coordinates
+
+# creating Gridder object
+coords = coordinates.GridCoords(cell_size=cell_size, npix=512)
+gridder = gridding.Gridder(
+    coords=coords,
+    uu=uu,
+    vv=vv,
+    weight=weight,
+    data_re=data.real,  # separating the real and imaginary values of our data
+    data_im=data.imag,
+)
+
+# -
+
+#
+# ### Getting the Dirty Image and Creating the Model
+#
+# First, we are going to get the dirty image from our Gridder object. We will use the Briggs weighting scale and set `robust=0.0` here as this option leads to a dirty image resembling the DSHARP CLEAN image (see [Part 1](HD143006_Part_1.html)).
+
+import torch
+
+img, beam = gridder.get_dirty_image(weighting="briggs", robust=0.0, unit="Jy/arcsec^2")
+# taking the dirty image and making it a tensor
+dirty_image = torch.tensor(img.copy())
+
+# We now have everything from the last tutorial loaded and we can create the model.
+
+from mpol.precomposed import SimpleNet
+
+model = SimpleNet(coords=coords, nchan=gridder.nchan)
+
+# ## Initializing Model with the Dirty Image
+#
+# We now have our model and data, but before we set out trying to optimize the image we should create a better starting point for our future optimization loops. A good idea for the starting point is the dirty image, since it is already a maximum likelihood fit to the data. The problem with this is that the dirty image contains negative flux pixels, while we impose the requirement that our sources must have all positive flux values. Our solution then is to optimize the RML model to become as close to the dirty image as possible (while retaining image positivity).
```

Comment:
A "better" starting point relative to what? A blank/uniform image? I think we could make this clearer by saying we need to pick a starting point, which could be basically anything, and the dirty image is a logical choice. Rather than the current language which sort of implies we are changing the existing starting point to the dirty image instead.

Reply:
I also think we should be careful with the language used at the end of this line. The current text says "Our solution then is to optimize the RML model to become as close to the dirty image as possible (while retaining image positivity)" which is true in this case; we're only enforcing image positivity which is a very simply prior, but in most cases we're not trying to make an image as close to the dirty image as possible, we're trying to make a better image. I think it would be useful to rephrase either emphasizing that image positivity is the only prior being considered here, or that we are trying to make the best possible image given the dirty image and the need for positive pixel values.

Reply:
@briannazawadzki Are there cases when we don't initialize the model with the dirty image before beginning optimization for the best image? My understanding was that this section of the tutorial is initializing the model to the dirty image while retaining the image positivity prior, so we would want the dirty image to closely resemble the model at this point. Then later we take this as the starting point and begin to perform the actual optimization such that the model fits the visibilities but also takes into account any other priors so the model resembles what we'd expect the true image to be.

Reply:
@hgrzy Oops, I'm sorry, I must not have my GitHub notifications configured correctly. Yes, your understanding is correct. The dirty image is being used as the starting point for the optimization loop. But sure, there could be cases where you don't want to do that. For example, you might want to see if you can still converge on a reasonable solution using the same priors but starting with a uniform blank image. It would probably take more iterations to converge than using the dirty image, since presumably the dirty image is already in the ballpark of what the final result will be, while a blank image needs a lot more adjusting.

My comment was mainly just being nitpicky about language for clarity.

Reply:
@briannazawadzki No worries! Ah that makes sense. Thanks for bringing up that good point.